docs/explanation/special-files.md

```diff
@@ -8,8 +8,6 @@ There are a few special files that React Router looks for in your project. Not a
 
 ## react-router.config.ts
```

Comment:
I kind of wanted to delete this file, but I didn't want to break existing links to it. I figured whittling down the content and making it mostly a reference file was the best way to go

Reply:
yeah maybe also add a `.server/.client files` page to the new `Framework Conventions` section and then you could hide the file from the sidebar and trim it down to the minimal set of links?

```
The docs in this file have been moved elsewhere:

- [root.tsx](../path/to/new/root.tsx)
- ...
- ```

Reply:
Okay, updated! Let me know if you have any further thoughts on those 2 files